megatron/training.py

```diff
@@ -40,7 +40,7 @@
 from megatron.utils import calc_params_l2_norm
 from megatron.core.pipeline_parallel import get_forward_backward_func
 from megatron.utils import report_memory, throughput_calculator, checkpoint_throughput_calculator
-from megatron.model.vision.knn_monitor import compute_feature_bank
+# from megatron.model.vision.knn_monitor import compute_feature_bank
```

Comment:
@YizhouZ why this import is commented? Please revert this change if this is a mistake. Thanks.

Reply:
sorry about this mistake. a new PR is provided to revert this https://github.com/microsoft/Megatron-DeepSpeed/pull/165. please review it.

Reply:
Thank you!